scripts/local_dev.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -ex
+
+MNT_PATH="database"
+PLUGIN_NAME="vault-plugin-database-elasticsearch"
+PLUGIN_CATALOG_NAME="vault-plugin-database-elasticsearch"
+
+#
+# Helper script for local development. Automatically builds and registers the
+# plugin. Requires `vault` is installed and available on $PATH.
+#
+
+# Get the right dir
+DIR="$(cd "$(dirname "$(readlink "$0")")" && pwd)"
+
+echo "==> Starting dev"
+
+echo "--> Scratch dir"
+echo "    Creating"
+SCRATCH="$DIR/tmp"
+mkdir -p "$SCRATCH/plugins"
+
+echo "--> Vault server"
+echo "    Writing config"
+tee "$SCRATCH/vault.hcl" > /dev/null <<EOF
```

Comment:
nit: Why not use `-dev-plugin-dir`?

Reply:
Good idea, changed in 507cc5319bdb4adbd706ad866cd591d087de9dfc